### ISSUE TITLE
difference between Maṣḥafa gǝṣṣāwe and gǝṣṣǝw

### ISSUE THREAD

**Issue by thea-m** [BetaMasaheft/Documentation#2461]
As pointed out by @nafisa-valieva , the difference between the records LIT1932Mashaf and LIT5654MGessew (used for instance in ESmy022) does not become clear from the records. A difference is encoded in relations, so they seem not to be doublets.. 
I found no issues or PRs regarding the creation LIT5654MGessew  (maybe it was before the new workflow). Does any of you spontaneously remember its characteristics? If so, it would be very helpful if the difference to the other Gǝṣṣāwe were explained in the abstract.
@eu-genia @DenisNosnitsin1970

**Comment by DenisNosnitsin1970** [comment]
You are right. Mashafa gessew and Mashafa gessawe, as far as I understand, are somewhat similar in structure, but I believe they are different. Gessawe appears to me a "generic title" (as the idea is only to collect beginnings of the readings according to the calendar), M. gessew may refer to a specific work with clearer boundaries. SDGM-007 uses the title Mashafa gessew. Among Ethio-Spare mss there are many M. gessew and less M. gessawe, otherwise it is vice versa :) (EAE does not refer to M. gessew). M. gessew descriptions are well developed and the work looks differently from M. gessawe. Today, there are also some church editions which can be consulted. Someone may have a look at the issue again and write smth in the abstract.

**Comment by nafisa-valieva** [comment]
@thea-m, I think this issue can be closed :-)

**Comment by thea-m** [comment]
@DenisNosnitsin1970 is responsible for this issue.

**Comment by nafisa-valieva** [comment]
> [@DenisNosnitsin1970](https://github.com/DenisNosnitsin1970) is responsible for this issue.

I see! Sorry!
